docs/api-reference/cli.md

```diff
@@ -106,6 +106,22 @@ PORT=4000 npx next start
 
 > Note: `PORT` can not be set in `.env` as booting up the HTTP server happens before any other code is initialized.
 
+### Working with Load-balancer Timeouts
+
+When deploying Next.js behind an upstream proxy (e.g. a load-balancer like AWS ELB/ALB) it's important to configure Next's underlying http-server with [keep-alive timeouts](https://nodejs.org/api/http.html#http_server_keepalivetimeout) that are _smaller_ than the upstream proxies' timeouts. Otherwise, once a keep-alive timeout is reached for a given TCP connection, Node.js will immediately terminate that connection without notifying the upstream proxies. This results in the upstream proxies throwing intermittent 5xx errors whenever they attempt to reuse a connection that Node.js has already terminated.
+
+To configure the timeout values for the production Next.js server, pass `--keepAliveTimeout` (in milliseconds) to `next start`, like so:
+
+```bash
+npx next start --keepAliveTimeout 70000
+```
+
+In some earlier versions of Node.js, there is [a regression that requires that `headersTimeout` is set to a value greater than the `keepAliveTimeout` value](https://github.com/nodejs/node/issues/27363). To configure [`headersTimeout`](https://nodejs.org/api/http.html#serverheaderstimeout) for the production Next.js server, pass `--headersTimeout` (in milliseconds) to `next start`, like so:
+
+```bash
+npx next start --keepAliveTimeout 70000 --headersTimeout 75000
```

Comment:
Should this be configured in `next.config.js` instead?

https://nextjs.org/docs/api-reference/next.config.js/disabling-http-keep-alive

Reply:
I thought about it but in [this thread](https://github.com/vercel/next.js/discussions/28642#discussioncomment-1805011) someone asked if these settings could be environment variables and @timneutkens made a good point about how that wouldn't make sense — since the changes only apply to `next start`. I just assumed the same logic would apply to `next.config.js`. Let me know if you think different.